app/views/albums/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row align-show">
+  <section class="small-6 columns">
+    <h3><%= @album.title%></h3>
+    <h4> Recorded by: <%= @album.recorded_by %> </h4>
+    <h5> Upvotes: <%= @album.ranked %> </h5>
+    <p> <%= @album.description %> </p>
+  </section>
+
+  <section class="small-6 columns">
+    <ul class="align-buttons">
+      <li><%= button_to 'Upvote', upvote_path(params[:id]), :class => "button", method: :put %></li>
```

Comment:
Since we already have the correct Album to upvote in the `@album` variable, we can actually just say `upvote_path(@album.id)` here instead.